Source/Core/Runtime/Utils/DebugUtils.cs

```diff
@@ -0,0 +1,46 @@
+using UnityEngine;
+
+namespace VRBuilder.Core.Utils
+{
+    /// <summary>
+    /// Utility class for debug tools.
+    /// </summary>
+    public static class DebugUtils
+    {
+        /// <summary>
+        /// Draws a custom wire cylinder gizmo.
+        /// </summary>
+        public static void DrawWireCylinderGizmo(Vector3 startPoint, Vector3 endPoint, float width, Color color)
+        {
```

Comment:
Suggesting to do a sanity check to not get exceptions.
```suggestion
        {
            if (width <= 0 || startPoint == endPoint)
            {
Debug.LogWarning($"Invalid parameters for DrawWireCylinderGizmo. Width: {width}, StartPoint: {startPoint}, EndPoint: {endPoint}. Width must be greater than zero and startPoint must not equal endPoint.");
                return;
            }
```

Reply:
Done but only for width < 0. It's fine to create a zero height cylinder imo, it's just a circle.